.github/workflows/builds.yml

```diff
@@ -94,3 +94,46 @@ jobs:
             COMMIT_HASH=${{ github.sha }}
             REVISION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.revision'] }}
             ENTERPRISE_TOKEN=${{ secrets.ENTERPRISE_TOKEN }}
+  docker-suppression-backup-service:
```

Comment:
It doesn't seem like the correct Dockerfile is being used [here](https://github.com/rudderlabs/rudder-server/actions/runs/4416443796/jobs/7740738131)